Argus-Labs/world-engine
# chore: fix issue with 'make rollup' leaving celestia-devnet container running

Closes: WORLD-880


## Overview

- On `make rollup` action, the `celestia_devnet` container leaves running after the interrupt action (ctrl+c) because it started on a separate action (`scripts/start-celestia-devnet.sh`) using `--detach` mode.

## Brief Changelog

- makefiles: use bash `trap` command, to catch exit and run `docker compose down chain celestia-devnet` to properly remove the `celestia_devnet` container & compose networks.

## Testing and Verifying

- run: `make rollup`
- interrupt & exit the running process with ctrl+c
- check `docker ps` and verify there's no `celestia_devnet` container running

![image](https://github.com/Argus-Labs/world-engine/assets/29672212/c8a2f7d6-9ad9-4709-8412-d08d0715dc8e)
^^ last log shows it properly remove both evm_base_shard & celestia_devnet container and compose networks.
